language/tools/vm-genesis/src/lib.rs

```diff
@@ -750,6 +766,22 @@ fn verify_genesis_write_set(
     );
 }
 
+/// Generate a dummy genesis `ChangeSet` that allows open publishing. For testing only.
+pub fn generate_dummy_genesis_change_set() -> ChangeSet {
```

Comment:
I personally find the name "dummy" confusing, but maybe it's just me. This is still the very same genesis we use for the network, but with extra capabilities.